docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/ddd-oriented-microservice.md

```diff
@@ -33,15 +33,15 @@ For example, an entity could be loaded from the database. Then part of that info
 
 Additionally, you need to have always-valid entities (see the [Designing validations in the domain model layer](#designing-validations-in-the-domain-model-layer) section) controlled by aggregate roots (root entities). Therefore, entities should not be bound to client views, because at the UI level some data might still not be validated. This is what the ViewModel is for. The ViewModel is a data model exclusively for presentation layer needs. The domain entities do not belong directly to the ViewModel. Instead, you need to translate between ViewModels and domain entities and vice versa.
 
-When tackling complexity, it is important to have a domain model controlled by aggregate roots (we go into this in more detail later) that make sure that all the invariants and rules related to that group of entities (aggregate) are performed through a single entry point or gate, the aggregate root.
+When tackling complexity, it is important to have a domain model controlled by aggregate roots (we go into this in more detail later) that make sure that all the invariants and rules related to that group of entities (aggregate) are performed through a single entry-point or gate, the aggregate root.
```

Comment:
it is -> it's 

remove (we go into this in more detail later). If you want, you can point people to where they could find those details on a second setence. Such as "For more information about x, see link."

Reply:
👍